### PR TITLE
Create portfolio2

### PR DESCRIPTION
**Pull request by Berucan**
I setup the environment for the project, added linters files, added index file, style file and modified README file, added project description , author details.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+            <li>javascript</li>
+          </ul>
+          <button type="button" class="btn-tonic">See Project</button>
+          </div>
+          </div>
+          </section>
+            <!--ABOUT MYSELF SECTION-->
+          <section class="about">
+              <div class="Description">
+            <h1 class="heading">About <br/> myself</h1>
```
Comment (Deepakdanger) [inline index.html]: - [x] `h1` tag can only use at only one time on each page. Kindly change it to the `h2 or h3` tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+            <p class="profile">
+              I’m a software developer! I can help you build a product , feature or
+              website Look through some of my work and experience! If you like what
+              you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <p class="info">LET'S CONNECT</p>
```
Comment (Deepakdanger) [inline index.html]: - [x]  Here, for `Lets connect` should be in `h2` or any other `hx` tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+                            <img src="images/github.png" alt="Github"></a>
+                    </li>
+                    <li class="icon">
+                        <a href="#">
+                          <img src="images/fingers.png" alt="Finger">
+                        </a>
+                    </li>
+                </ul>
+            </div>
+              <input type="button" class="btn-tonic" value="Get my resume">
```
Comment (Deepakdanger) [inline index.html]: - [x] Here the requirement is :-
- The button (e.g., "Get my resume") is a link a or button tag, and follow the button styling: Active, Hover and Pressed (see design)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="button" class="btn-tonic">See Project</button>
+          </div>
+          </div>
+          </section>
+            <!--ABOUT MYSELF SECTION-->
+          <section class="about">
```
Comment (Deepakdanger) [inline index.html:169]: - [x] The background color & border-style should be the same as the original website.
- your website
edit2
- Original website
o2

File: index.html [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+              <input type="button" class="btn-tonic" value="Get my resume">
+            </div>
+
+            <div class="language-icon">
+            <div class="language-title">
+            <p>Language</p>
+            <i class="fas fa-chevron-right"></i>
+            </div>
+            <div class="language">
+              <ul class="language-list">
```
Comment (Deepakdanger) [inline index.html:216]: - [x] Here the design should same as the original website.
- your website
edit1
- Original website
original1

File: index.html [hunk 6]
```diff
@@ -0,0 +1,235 @@
...
+                    <a href="#">
+                      <img src="images/fingers.png" alt="Finger">
+                    </a>
+                </li>
+            </ul>
+        </div>
+    </section>
+
+          <!--WORK section-->
+          <section class="work">
```
Comment (Deepakdanger) [inline index.html:65]: -[ ] Here work section background color, border, the margin should match the original website.
- your website
edit3

- Original website  
o3

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Berucan , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by lily-coder — COMMENTED** [review]
Hi @Berucan , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linter errors. Please fix the linter errors and submit for another review

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Review by Hamzaoutdoors — APPROVED** [review]
Hi @Berucan, 

## Congratulations! 🎉 Project Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀

To Highlight:

- [x] The Pull Request has a proper title and description. ✔️
- [x] You have added a descriptive Readme file. Good Job. ✔️
- [x] There is no linter error is present. ✔️
- [x] Project has met all the technical aspects you needed for this project! ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL] 
You have broken links in your README such as `Live demo` and `MIT` links, so Kindly try to address that, because keeping broken links in the readme file can be unprofessional

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**